cmd/kube-apiserver/app/server.go

```diff
@@ -268,6 +269,8 @@ func Run(s *options.APIServer) error {
 		glog.Fatalf("Failed to initialize plugins: %v", err)
 	}
 
+	mux := http.NewServeMux()
```

Comment:
I guess this initialization can go away, the default case for `RestfulContainer` should be enough.Trying to remember why I did this in the first place.